types/src/sender.d.ts

```diff
@@ -70,7 +73,7 @@ export class Sender {
      * Closes the connection to the database. <br>
      * Data sitting in the Sender's buffer will be lost unless flush() is called before close().
      */
-    close(): Promise<any>;
+    close(): Promise<void>;
```

Comment:
Return type is not consistent with the actual one which is boolean.

Reply:
yes, i generated this before i added the `return true;` line.
will regenerate everything, just in case.

Reply:
removed `return true;`, it is consistent now